src/interfaces/userRequest.ts

```diff
@@ -47,7 +46,8 @@ export interface SignUserRequest {
   meta: {
     isSignAction: true
     accountAddr: AccountId
-    networkId: NetworkId
+    chainId?: bigint
+    networkId?: NetworkId
```

Comment:
do not make the networkId optional. Pass it to the benzin and use only the chainId